.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -91,22 +87,6 @@ jobs:
           cmake --install ./Build
           cd -
 
-      - name: Build Kokkos kernels library
```

Comment:
Do we want to keep this for testing purposes, or is it being fully removed?

Reply:
It is still built and tested in `tests_linux.yml`.